Source/Engine/Core/Math/Float2.cs

```diff
@@ -1273,6 +1273,33 @@ public static Float2 TransformNormal(Float2 normal, Matrix transform)
             return result;
         }
 
+        /// <summary>
+        /// funcion for grid snaping snap to absolute world grid 
+        /// <br>desined for snaping using a ray / draging object</br>
+        /// </summary>
+        /// <param name="Translation"></param>
+        /// <param name="GridSize"></param>
+        /// <returns>out = Ceil(((A - (GridSize * 0.5f)) / GridSize) * GridSize</returns>
+        public static Float2 SnapToGrid(Float2 Translation, Float2 GridSize)
```

Comment:
Rename `Translation` to `translation` and `GridSize` to `gridSize` to match code style (all occurrences of that method in this PR).